server/processor/processor.go

```diff
@@ -77,7 +77,8 @@ func (p *Processor) HandleEvent(eventType string, data []byte) error {
 			p.PRCommentProcessor.Process(data)
 			return nil
 		}
-		return nil
```

Comment:
Why to remove this line of `return nil`?

Reply:
Because it (and the return statements above) is dup with https://github.com/pouchcontainer/pouchrobot/pull/13/files/7bcb37960d98cfc4de8280f3b0f6963c55986b86#diff-f0f11ac90b9bacc1a1fb8f873cd69dfcL84, IMO

WDYT?

Reply:
Yeah, you are correct.